pandas/core/frame.py

```diff
@@ -3994,8 +3989,6 @@ def _sanitize_column(self, value):
                     value = maybe_convert_platform(value)
                 else:
                     value = com.asarray_tuplesafe(value)
-            elif value.ndim == 2:
-                value = value.copy().T
```

Comment:
i think we stopped making a copy here?

Reply:
The final `else:` is still doing `value = value.copy()`, so that should not have changed I think

Reply:
never mind, it gets copied below